src/__tests__/modules/QueryBuilder.spec.ts

```diff
@@ -570,14 +570,14 @@ describe('QueryBuilder', () => {
         ).toThrow(QueryBuilderErrors.NOT_AN_ARRAY)
       })
 
-      it('should return null if an empty array is passed', () => {
+      it('should return an empty array if an empty array is passed', () => {
```

Comment:
In fact the function does not return an empty array, but a query which includes an empty array.